src/Workspaces/Core/Portable/Log/HistogramLogAggregator.cs

```diff
@@ -32,13 +32,19 @@ public HistogramLogAggregator(int bucketSize, int maxBucketValue)
         protected override HistogramCounter CreateCounter()
             => new(_bucketSize, _maxBucketValue, _bucketCount);
 
-        public void IncreaseCount(object key, decimal value)
+        public void IncreaseCount(TKey key, int value)
         {
             var counter = GetCounter(key);
             counter.IncreaseCount(value);
         }
 
-        public HistogramCounter? GetValue(object key)
+        public void IncreaseCount(TKey key, TimeSpan timeSpan)
```

Comment:
This name is a little confusing, especially as an overload to another method that takes an int.. I wonder if it makes sense to rename it, and/or make it an extension method?

Don't know

Reply:
Yep, absolutely makes sense.

Reply:
Renamed the new method to "LogTime"; I suspect the old name was because the naming was stolen from the thing I renamed to CountLogAggregator. I'll do some further cleanup here later.